caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -38,7 +38,7 @@ class ChartContainer extends React.Component {
 
       width: () => this.chartContainerRef.getBoundingClientRect().width,
 
-      height: () => parseInt(this.props.height, 10) - 100,
+      height: () => parseInt(this.props.height, 10),
```

Comment:
I'm not sure why we do -100 here, the heatmap was squashed in y direction with -100 in height.